<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MovieListViewWidget</class>
 <widget class="QWidget" name="MovieListViewWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="listPlaceholderButton">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>100</y>
     <width>141</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>List placeholer</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addMovieButton">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>170</y>
     <width>83</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Add movie</string>
   </property>
  </widget>
  <widget class="QPushButton" name="goBackButton">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>210</y>
     <width>83</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Go back</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
